.env.example

```
@@ -0,0 +1,3 @@
+﻿# GitHub API token for live stats and Good First Issues (required)
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# GitHub API token for live stats and Good First Issues (required) key has incorrect delimiter

(IncorrectDelimiter)